owl/EASE-OBJ.owl

```diff
@@ -562,9 +560,9 @@
     <owl:ObjectProperty rdf:about="http://www.ease-crc.org/ont/EASE-OBJ.owl#isPhysicallyContainedIn">
         <rdfs:subPropertyOf rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#isLocationOf"/>
         <rdfs:domain rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#Entity"/>
-        <rdfs:range rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#Entity"/>
-        <rdfs:comment>A spatial relation holding between an object (the container), and objects it contains.</rdfs:comment>
-        <rdfs:label>is physically contained in</rdfs:label>
+		<rdfs:range rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#Entity"/>
```

Comment:
it is not so nice having this diff here.
It seems you have changed the indentation, or?
Try to change it back and the diff should disappear here.

Reply:
Should be corrected.